tests/unit/src/test_conditionsStore.cpp

```diff
@@ -468,7 +444,7 @@ SCENARIO( "Providing derived conditions", "[ConditionStore][DerivedConditions]"
 			THEN( "read-only prefixed provider should reject erase" )
 			{
 				mockProvPrefixA.SetROPrefixProvider(store, "prefixA: ");
-				REQUIRE_FALSE( store.Erase("prefixA: test") );
+				REQUIRE_FALSE( store.Set("prefixA: test", 0) );
```

Comment:
As above, this is testing "Set" functionality, not erasure.

Reply:
Agreed, I updated the description to talk about setting.